x/wasm/keeper/wasmtesting/mock_keepers.go

```diff
@@ -118,3 +118,16 @@ func (m MockIBCTransferKeeper) GetPort(ctx sdk.Context) string {
 	}
 	return m.GetPortFn(ctx)
 }
+
+var _ types.WormholeKeeper = &MockWormholeKeeper{}
+
+type MockWormholeKeeper struct {
+	HasInstantiateAllowlistFn func(ctx sdk.Context, contract string, codeId uint64) bool
+}
+
+func (m MockWormholeKeeper) HasInstantiateAllowlist(ctx sdk.Context, contract string, codeId uint64) bool {
```

Comment:
Are there any tests that actually use this, yet?

Reply:
No. I can add tests as part of a follow up PR.